system/jlib/jlog.hpp

```diff
@@ -849,6 +849,7 @@ constexpr LogMsgCategory MCdebugInfo(MSGAUD_programmer, MSGCLS_information, Debu
 constexpr LogMsgCategory MCauditInfo(MSGAUD_audit, MSGCLS_information, AudMsgThreshold);
 constexpr LogMsgCategory MCstats(MSGAUD_operator, MSGCLS_progress, ProgressMsgThreshold);
 constexpr LogMsgCategory MCoperatorInfo(MSGAUD_operator, MSGCLS_information, InfoMsgThreshold);
+constexpr LogMsgCategory MCmetrics(MSGAUD_operator, MSGCLS_metric, CriticalMsgThreshold);
```

Comment:
minor:  It doesn't make too much difference... I'm not sure what detail level it should be, but critical seems wrong.  ErrMsgThreshold might be more appropriate.

Reply:
Changed to ErrMsgThreshold per suggestion. I used critical prior to ensure that metrics were logged regardless.